server/sonar-server/src/main/java/org/sonar/server/project/ws/CreateAction.java

```diff
@@ -78,8 +79,7 @@ public void define(WebService.NewController context) {
 
     action.setChangelog(
       new Change("6.3", "The response format has been updated and does not contain the database ID anymore"),
-      new Change("6.3", "The 'key' parameter has been renamed 'project'")
-    );
+      new Change("6.3", "The 'key' parameter has been renamed 'project'"));
```

Comment:
The parameter is documented as `.setSince("6.4")`. No Changelog entry is required in those cases.